src/index.js

```diff
@@ -575,10 +575,10 @@ export class ULabel {
             }
         }, {"signal": signal});
 
-        window.addEventListener("beforeunload", function (e) {
+        window.addEventListener("beforeunload.ulabel", function (e) {
             var confirmationMessage = '';
             if (ul.state["edited"]) {
-                confirmationMessage = 'You have made unsave changes. Are you sure you would like to leave?';
+                confirmationMessage = 'You have made unsaved changes. Are you sure you would like to leave?';
                 (e || window.event).returnValue = confirmationMessage; //Gecko + IE
                 return confirmationMessage; //Gecko + Webkit, Safari, Chrome etc.
             }
```

Comment:
Tried to test this, it appears that [all browsers have dropped support for custom messages](https://stackoverflow.com/a/38880926).

We still need something to trigger the message, see [this](https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeunload_event) for a couple options. 
I've tried all of these and they work fine (sometimes) in Firefox and Chrome:
```javascript
window.addEventListener(
    "beforeunload",
    (event) => {
        if (ul.state["edited"]) {
            (event || window.event).returnValue = 1;
        }
    },
    { "signal": signal },
);
```
Both `window.event` and `event.returnValue` say they are deprecated, but this was the only non-convoluted way I got this to work.

Alternatively:
```javascript
window.onbeforeunload = function (e) {
    return ul.state["edited"]
}
```
not sure if the `onbeforeunload` property introduces some issues, so I tried with JQuery as well:
```javascript
$(window).on("beforeunload", function (e) {
    return ul.state["edited"];
});
```
but this appeared to inconsistently raise the popup even when no edits were made. 
Might be best to just leave it as-is and/or remove the message for now.

Reply:
Actually I think both of the last two examples inconsistently raise the pop up in the non-edited state. Maybe returning a purely truthy value has some weird inconsistency?

Reply:
Yeah changing either of the later 2 examples to:
```javascript
if (ul.state["edited"]) {
    return 1;
}
```
doesn't have any inconsistencies :upside_down_face:

Reply:
`return false;` also gets the inconsistent behavior... returning anything at all triggers it!
`return ul.state["edited"] ? 1 : null` triggers it!
`return ul.state["edited"] ? 1 : undefined` doesn't, as that's what functions will return if they have no return value. That must be what the caller checks for when running this, at least in some cases.

bonkers

Reply:
[a6700e8](https://github.com/SenteraLLC/ulabel/pull/160/commits/a6700e82d8efeb0d127bee9d358ad99a1d075091)